src/api/posts.ts

```diff
@@ -0,0 +1,25 @@
+import { Post } from '../react-app-env';
+
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getPosts(userId: string): Promise<Post[]> {
+  const response = await fetch(`${BASE_URL}/posts?userId=${userId}`);
+
+  if (!response.ok) {
```

Comment:
Here you could use a try-catch block to handle the asynchronous request, making the code more readable